saleor/graphql/account/schema.py

```diff
@@ -169,8 +172,8 @@ def resolve_staff_users(self, info, query=None, **kwargs):
     @one_of_permissions_required(
         [AccountPermissions.MANAGE_STAFF, AccountPermissions.MANAGE_USERS]
     )
-    def resolve_user(self, info, id):
-        return resolve_user(info, id)
+    def resolve_user(self, info, id=None, email=None):
+        return resolve_user(info, id, email)
```

Comment:
We have a `validate_one_of_args_is_in_query` validator which is used in queries like this to check if any of the two params was passed, as it is required to pass one of them.

Reply:
Yep thanks, I actually did the same in attributes but tbh, I totally forgot about the existence of this PR, welcome back! I'll fix everything then commit.